scout/adapter/mongo/clinvar.py

```diff
@@ -68,15 +64,9 @@ def delete_submission(self, submission_id):
         # return deleted_count, deleted_submissions
         return deleted_objects, deleted_submissions
 
-    def get_open_clinvar_submission(self, institute_id):
-        """Retrieve the database id of an open clinvar submission for an institute,
-        if none is available then create a new submission and return it
-
-        Args:
-             institute_id(str): an institute ID
-
-        Returns:
-             submission(obj) : an open clinvar submission object
+    def get_open_clinvar_submission(self, institute_id: str, user_id: str) -> dict:
```

Comment:
I can't quite remember, but wasn't `dict` among the type hints that only worked for more recent python, and we should use typing `Dict` instead?

Reply:
Hehe that's confusing I know. It's like this, if you have only `dict` , then it works, because it's one of the "native" data types of python. But if you have dict[str, str] then it will crash on python 3.8. You'd need to write Dict[str, str]